internal/rsynctest/rsynctest.go

```diff
@@ -105,7 +105,7 @@ func New(t *testing.T, modules []rsyncd.Module, opts ...Option) *TestServer {
 			}
 
 			if err != nil {
-				log.Print(err)
+				log.Printf(err.Error())
```

Comment:
This should be `log.Printf("%s", err.Error())`, otherwise the error message will be used as format string.

Reply:
done, PTAL